convert/src/label_studio_to_badgerdoc/models/label_studio_models.py

```diff
@@ -85,18 +85,18 @@ class Prediction(BaseModel):
 
 
 class Annotation(BaseModel):
-    id: int = 7
+    id: int = 1
     completed_by: int = 1
     result: List[ResultItem]
     was_cancelled: bool = False
     ground_truth: bool = False
-    created_at: str = "2022-12-13T09:57:08.451845Z"
-    updated_at: str = "2022-12-13T09:57:08.451875Z"
-    lead_time: float = 121.434
+    created_at: str = ""
+    updated_at: str = ""
```

Comment:
According to this these values should not be hardcoded with default values:
https://github.com/epam/badgerdoc/pull/117#discussion_r1062214735

Reply:
Fixed